docker-compose.yml

```diff
@@ -156,6 +156,29 @@ services:
       - "traefik.frontend.rule=${TRAEFIK_FRONTEND_RULE:-}PathPrefix:/bety/"
       - "traefik.backend=bety"
 
+  # ----------------------------------------------------------------------
+  # Pecan-RStudio
+  # ----------------------------------------------------------------------
+  rstudio:
+    image: pecan/base:${PECAN_RSTUDIO_VERSION:-latest}
+    restart: unless-stopped
+    networks:
+      - pecan
+    environment:
+      - USER=${PECAN_RSTUDIO_USER:-carya}
+      - PASSWORD=${PECAN_RSTUDIO_PASS:-ilinois}
+    depends_on:
+      - bety
+    entrypoint: /init
+    labels:
+      - "traefik.enable=true"
+      - "traefik.backend=rstudio"
+      - "traefik.port=8787"
+      - "traefik.frontend.rule=${TRAEFIK_FRONTEND_RULE:-}PathPrefix:/rstudio"
+      - "traefik.website.frontend.whiteList.sourceRange=${TRAEFIK_IPFILTER:-172.16.0.0/12}"
+    ports:
```

Comment:
remove ports completely. If you want to open the port you can do it in docker-compose.override.yml.

Reply:
Ports are no more